<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FindSermon</class>
 <widget class="QDialog" name="FindSermon">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>543</width>
    <height>281</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Find Sermon</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>35</x>
     <y>74</y>
     <width>471</width>
     <height>128</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="sermonFields_gridLayout">
    <item row="4" column="3">
     <widget class="QLineEdit" name="description_lineEdit"/>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="speaker_lbl">
      <property name="text">
       <string>Speaker:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="title_lbl">
      <property name="text">
       <string>Title:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="date_lbl">
      <property name="text">
       <string>Date:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QLineEdit" name="location_lineEdit"/>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="location_lbl">
      <property name="text">
       <string>Location:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLabel" name="description_lbl">
      <property name="text">
       <string>Description</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QLineEdit" name="speaker_lineEdit"/>
    </item>
    <item row="3" column="3">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="from_label">
        <property name="text">
         <string>From:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="from_dateEdit">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="to_label">
        <property name="text">
         <string>To:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="to_dateEdit">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="3">
     <widget class="QLineEdit" name="title_lineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="directions_label">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>24</y>
     <width>481</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600; font-style:italic; color:#005500;&quot;&gt;Please enter your search criteria below.&lt;br/&gt;Your results will appear in the Main Table.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="clearSearch_pushButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>226</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Clear Search</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
